Frameworks/include/CGPatternInternal.h

```diff
@@ -58,5 +42,14 @@ CGRect _CGPatternGetFinalPatternSize(CGPatternRef pattern);
 */
 bool _CGPatternIsColored(CGPatternRef pattern);
 
+COREGRAPHICS_EXPORT CGPatternRef _CGPatternCreateColorspace(void* info,
```

Comment:
no longer needed.

---
In reply to: [109017103](https://github.com/Microsoft/WinObjC/pull/2374#discussion_r109017103) [](ancestors = 109017103)